src/gc/env/gcenv.os.h

```diff
@@ -391,6 +393,14 @@ class GCToOSInterface
     // Return:
     //  Number of processors on the machine
     static uint32_t GetTotalProcessorCount();
+
+    static bool CanEnableGCNumaAware();
+    static bool GetNumaProcessorNodeEx(PPROCESSOR_NUMBER proc_no, uint16_t *node_no);
+
+    static bool CanEnableGCCPUGroups();
+    static uint16_t GetNumActiveProcessors();
```

Comment:
I do not think we need a special API for this. This should be just a part of `GetCurrentProcessCpuCount` - the check for enable CPU groups should be inside GetCurrentProcessCpuCount.